common/workunit/workunit.cpp

```diff
@@ -13648,7 +13650,11 @@ void executeThorGraph(const char * graphName, IConstWorkUnit &workunit, const IP
         VStringBuffer job("%s-%s", wuid.str(), graphName);
         runK8sJob("thormaster", wuid, job, queryComponentConfig().getPropBool("@deleteJobs", true), { { "graphName", graphName} });
     }
-
+    unsigned __int64 blockedTime = elapsedTimer.elapsedNs();
```

Comment:
@shamser I think this timing the execution of the thor job, rather than the blocked time.
Also, Jake is better placed to review the code.

Reply:
@jakesmith Please can you review.

Reply:
I've removed this code.  I'm looking at how blocked time may be tracked in container environment.